### PR TITLE
CBL-3528 Update release note for 3.0.2

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

**Comment by RayOffiah** [comment]
Ready for review

File: modules/ROOT/pages/_partials/commons/common-releasenotes.adoc [hunk 1]
```diff
@@ -14,33 +14,68 @@ include::{root-partials}_show_page_header_block.adoc[]
...
 :param-3-0-0:
 :param-3-0-1:
+:param-3-0-2:
 endif::[]
 //
 // END RELEASE SELECTION BLOCK -- Decide which releases are relevant for a given platform
 
 
+// BEGIN RELEASE BLOCK -- Release 3-0-2 (All) -- August 2022
+// Clone this block for each maintenance release and edit release nos as required
```
Comment (maria-robobug) [inline modules/ROOT/pages/_partials/commons/common-releasenotes.adoc]:
```suggestion
// Clone this block for each maintenance release and edit release notes as required
```

File: modules/android/pages/_partials/pn-issues-list.adoc [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+
+* https://issues.couchbase.com/browse/CBL-3274[CBL-3274] -- https://issues.couchbase.com/browse/CBL-3274[Expose the IndexConfiguration class]
+
+* https://issues.couchbase.com/browse/CBL-3084[CBL-3084] -- https://issues.couchbase.com/browse/CBL-3084[Make SSLException a recoverable error in AbstractCBLWebSocket]
+
+* https://issues.couchbase.com/browse/CBL-2999[CBL-2999] -- https://issues.couchbase.com/browse/CBL-2999[Implement enhanced pinned server certificate feature]
+
+// end::Enhancements-3-0-2[]
+
+== KI
```
Comment (maria-robobug) [inline modules/android/pages/_partials/pn-issues-list.adoc:43]: Took me a second to understand what this was, would it not be better to say it explicitly: `Known Issues`?
Or is it something else?

**Review by maria-robobug — COMMENTED** [review]
👍 LGTM, just some minor comments.
Though I don't know much about couchbase-lite, so will probably need another pair of eyes in terms of the actual features going in.

File: modules/android/pages/_partials/pn-issues-list.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/android/pages/_partials/pn-issues-list.adoc:43]: I was puzzled by this one, too, as on the page as html, it's _Known Issues_ - I followed a trail of includes four-deep through different _partials directories before I gave up.
Suffice to say mobile docs are somewhat complex.

Comment (RayOffiah) [reply modules/android/pages/_partials/pn-issues-list.adoc:43]: Yes, I'm going to do a quick document to explain how this works.

The `== KI` is just a placeholder to show where you fill in the Known Issues; the heading doesn't actually show up when the pages are compiled.

File: modules/ROOT/pages/_partials/commons/common-releasenotes.adoc [hunk 3]
```diff
@@ -14,33 +14,68 @@ include::{root-partials}_show_page_header_block.adoc[]
...
+==== Fixed Issues
+include::{module-partials}pn-issues-list.adoc[tag=Fixed{param-release-tag}]
+
+
+==== Known Issues
+
+include::{module-partials}pn-issues-list.adoc[tag=KI{param-release-tag}]
+
+endif::param-3-0-2[]
+// END RELEASE BLOCK -- Release 3-0-2 (All) -- August 2021
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-releasenotes.adoc:78]: typo: 2022

File: modules/objc/pages/_partials/pn-issues-list.adoc [hunk 4]
```diff
@@ -1,6 +1,19 @@
 = Release Note issues for Obj-C
 
+== 3.0.2
```
Comment (osfameron) [inline modules/objc/pages/_partials/pn-issues-list.adoc:3]: ObjC notes is missing "None for this release" for known issues

File: modules/c/pages/_partials/pn-issues-list.adoc [hunk 5]
```diff
@@ -1,5 +1,25 @@
 = Release Note issues for  C
 
+== 3.0.2
```
Comment (osfameron) [inline modules/c/pages/_partials/pn-issues-list.adoc:3]: C notes is missing "None for this release" for known issues

**Review by osfameron — APPROVED** [review]
Great work Ray! 👍 

Couple of minor comments inline, no blockers.

We should probably have a chat and go over whether we *want* to continue doing this in this abstracted style?

It doesn't feel like we're gaining a lot (as we still have a partial per toolkit, with repetition of ticket numbers, e.g. between Android and Java), so it might be simpler to make *new* release notes in future simply plain asciidoc text, with one .adoc per toolkit?

(But that's a discussion for another day)

**Comment by RayOffiah** [comment]
@osfameron

Hi 👋🏾.

Yes, I didn’t get the feeling there was any less work involved by breaking this up into so many conditional partials.

I think the plan was that this structure would make automation easier, but the script I ran into would need a bit of work (the directory structure seems to have changed) to get it working.

I agree that it would make it a lot easier for people if we just built each release note as a flat note without all the partials

OR

Build an Antora extension/ macro that we stick at the top of each release note:

release-note::3.0.2[c, java, ios, Android]

This would scan the jiras and build the release note for each component.

Either way, I agree that there doesn’t seem to be much of an advantage in carrying on with this structure. One discrete note per component would be easier to deal with.


________________________________

**Pull request merged**